src/common/test-runtime/project.json

```diff
@@ -9,7 +9,7 @@
     "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24813-03",
     "System.Linq.Parallel": "4.0.1",
     "OpenCover": "4.6.519",
-    "ReportGenerator": "2.5.0",
+    "ReportGenerator": "2.5.9",
```

Comment:
> we don't really use this in corefx any longer

So I can just update corefx and not build tools? Can all this be deleted from build tools then? I'm not clear on why it's in both places.